src/google/Common.js

```diff
@@ -144,8 +145,14 @@
         google.visualization.events.addListener(this._chart, "select", function () {
             var selectedItem = context._chart.getSelection()[0];
             if (selectedItem) {
-                context.click(context.rowToObj(context.data()[selectedItem.row]), context.columns()[selectedItem.column]);
+                context._selection = {
+                    data: context.rowToObj(context.data()[selectedItem.row]),
+                    column: context.columns()[selectedItem.column]
+                };
+            } else {
+                context._selection = {};
```

Comment:
@gordonsmith what structure should we use for multi selection? So I can add
to am charts

On Thursday, October 15, 2015, Gordon Smith <notifications@github.com>
wrote:

> In src/google/Common.js
> <https://github.com/hpcc-systems/Visualization/pull/1046#discussion_r42094503>
> :
>
> > @@ -144,8 +145,14 @@
> >          google.visualization.events.addListener(this._chart, "select", function () {
> >              var selectedItem = context._chart.getSelection()[0];
> >              if (selectedItem) {
> > -                context.click(context.rowToObj(context.data()[selectedItem.row]), context.columns()[selectedItem.column]);
> > +                context._selection = {
> > +                    data: context.rowToObj(context.data()[selectedItem.row]),
> > +                    column: context.columns()[selectedItem.column]
> > +                };
> > +            } else {
> > +                context._selection = {};
>
> If you change this to: context._selection = {data: {}, column: null, };
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcc-systems/Visualization/pull/1046/files#r42094503>.
>